profiler/internal/pprofutils/pprofutils.go

```diff
@@ -0,0 +1,12 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
Does this type really warrant its own file?

Reply:
No, but where else do you suggest to put it? It's referenced by 3 different files in this repo and none of them seem like a natural home.

```
$ rg ValueType
text.go
29:			SampleType: []*profile.ValueType{{
35:			PeriodType: &profile.ValueType{},
60:				p.SampleType = append(p.SampleType, &profile.ValueType{

delta_test.go
66:			deltaConfig := Delta{SampleTypes: []ValueType{{Type: "x", Unit: "count"}}}
80:			deltaConfig := Delta{SampleTypes: []ValueType{{Type: "foo", Unit: "count"}}}

delta.go
26:	SampleTypes []ValueType
```